cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,75 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/settings.pageObject.js';
+import SignInPageObject from '../support/pages/signIn.pageObject.js';
 
-  });
+const settingsPage = new SettingsPageObject();
+const SignInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Settings page', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.visit(settingsPage.url);
   });
 
   it('should provide an ability to update username', () => {
-
+    const newUserName = 'stepan_bandera';
+    settingsPage.usernameField
```

Comment:
```suggestion
    settingsPage.usernameField
```
wrap this and similar commands inside page object methods